environment.yml

```diff
@@ -6,19 +6,18 @@ channels:
   - conda-forge
   - defaults
 dependencies:
-  - conda-forge::r-data.table=1.11.4
-  - conda-forge::r-gplots=3.0.1
-  - bioconda::bioconductor-edger=3.22.5
-  - conda-forge::r-markdown=0.8
-  - bioconda::khmer=2.1.2
-  - bioconda::sourmash=2.0.1
+  - r-data.table=1.12.4
```

Comment:
https://github.com/nf-core/rnaseq/blob/master/environment.yml#L12 why did you remove the channels conda-forge, bioconda rnaseq still has that